### PR TITLE
[2.3] [Validator] spanish translation for invalid charset message

### PR DESCRIPTION
**Pull request by phansys**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

ref. #13528

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.es.xlf [hunk 1]
```diff
@@ -278,6 +278,10 @@
                 <source>This value should not be identical to {{ compared_value_type }} {{ compared_value }}.</source>
                 <target>Este valor no debería ser idéntico a {{ compared_value_type }} {{ compared_value }}.</target>
             </trans-unit>
+            <trans-unit id="73">
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: id is 80 now

Comment (phansys) [inline src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: Updated. Thank you @nicolas-grekas.

**Comment by javiereguiluz** [comment]
@phansys I'm not sure if this translation is 100% accurate. It now says: "this value doesn't match 'UTF-8'" and it should say "the charset of this value doesn't match UTF8" or "this value doesn't use the UTF8 charset".

**Comment by phansys** [comment]
@javiereguiluz, it's updated now based on your comments.

**Comment by javiereguiluz** [comment]
@phansys thanks!

:+1: the translation is now accurate.

**Comment by fabpot** [comment]
Thank you @phansys.

**Pull request closed without merging**